README.md

```diff
@@ -1 +1,26 @@
-# node-repl-lab
\ No newline at end of file
+# Running JavaScript Code in Node REPL
+
+## Objectives
+
+1. Use REPL
+1. Execute code in REPL
+1. Exit REPL
+
+## Introduction
+
+Understanding global and scoping will give you a good foundation for future development.
+
+## Instructions
+
+1. Enter REPL
+1. Write Node code to perform math by creating a sum function
```

Comment:
be more specific hear - create a function called `sum`. which adds two numbers together and returns the solution. does the function accept the numbers as parameters or as variables defined outside the function?